clickhouse_std.go

```diff
@@ -147,51 +148,71 @@ func (d *stdDriver) Open(dsn string) (_ driver.Conn, err error) {
 var ErrHttpNotSupported = errors.New("HTTP: not supported")
 
 type httpTransport struct {
```

Comment:
@ortyomka starting to wonder if we really need the transport - the implementations of the respective transports are identical. Could we just not interface the connection itself instead?

Reply:
Yeah, it is what I want to do next

Reply:
I started to do this, but it's not trivial for now. 

What should be in connect interface?
What to do with `release(connect*, error)`?
What to do with return values in PrepareContext? (for now there *batch and *httpBatch)

I have a some idea, will implement soon, but some structures will be changed to interfaces.

Reply:
`transport` has been removed. Now `stdConnect` represntive connection in std driver. Native driver will be updated later (next PR, I guess).